packages/appstore/src/components/modals/open-positions-svg-modal/open-positions-svg-modal.scss

```diff
@@ -0,0 +1,10 @@
+.open-positions-svg {
+    &__modal {
+        &-title {
+            margin-bottom: 2.4rem;
+        }
+        &-footer {
+            padding: 0 2.4rem 2.4rem !important;
```

Comment:
do we need important here

Reply:
yes, without it it will render default size ie, 16px. and figma specifies 24px.